htlcswitch/link.go

```diff
@@ -1079,35 +1059,11 @@ out:
 
 			break out
 
-		case <-l.logCommitTick:
```

Comment:
does this mean that in certain scenarios the commit dance will be faster?

Reply:
Yes!